### PR TITLE
Add main key to package.json

### PR DESCRIPTION
**Pull request by th3fallen**
Add a main key to package.json for webpack with package splitting

### CODE REVIEW

**Comment by th3fallen** [comment]
For those of us that wish to do webpack vendor splitting based on our package.json file since your package is named like a file without a main key webpack treats it as a file not a directory.

**Comment by ameech** [comment]
@chinchang any luck on getting this merged, I'm running into this issue as well. Thanks!

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "email": "chinchang457@gmail.com",
     "url": "http://kushagragour.in"
   },
+  "main": "hint.css",
```
Comment (chinchang) [inline package.json]: I made the change.

**Comment by chinchang** [comment]
merged manually. Thanks @th3fallen 👍 

@ameech its live :)

fixes #149

**Comment by ameech** [comment]
Thanks @chinchang!

**Pull request closed without merging**